system-tests/launchers/file-transfer-consumer/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2020, 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *       Fraunhofer Institute for Software and Systems Engineering - added dependencies
+ *
+ */
+
+plugins {
+    `java-library`
+    id("application")
+    id("com.github.johnrengelman.shadow") version "7.0.0"
+}
+
+val jupiterVersion: String by project
+val rsApi: String by project
+
+dependencies {
+    implementation(project(":core"))
+
+    implementation(project(":extensions:in-memory:assetindex-memory"))
+    implementation(project(":extensions:in-memory:transfer-store-memory"))
+    implementation(project(":extensions:in-memory:negotiation-store-memory"))
+    implementation(project(":extensions:in-memory:contractdefinition-store-memory"))
+
+    implementation(project(":extensions:api:observability"))
+
+    implementation(project(":extensions:filesystem:configuration-fs"))
+    implementation(project(":extensions:iam:iam-mock"))
+
+    implementation(project(":extensions:api:control"))
+
+    implementation(project(":data-protocols:ids"))
+
+    implementation(project(":samples:04.0-file-transfer:api"))
```

Comment:
Dependency to a sample? I would remove it to have a complete separated launcher.